map2csv/webtool_map2csv.py

```
@@ -0,0 +1,238 @@
+import streamlit as st
+import numpy as np
+import cv2
```

Ruff (0.11.9)

`cv2` imported but unused

Remove unused import: `cv2`

(F401)